play-services-nearby-core-ui/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (C) 2013-2017 microG Project Team
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="service_name_exposure">Notifiche di esposizione al virus</string>
+    <string name="pref_exposure_enable_info_summary">Per abilitare le notifiche di esposizione al virus, apri qualsiasi applicazioni che le supporti.</string>
+    <string name="prefcat_exposure_apps_title">Applicazioni che utilizzano le notifiche di esposizone al virus</string>
+    <string name="pref_exposure_collected_rpis_title">Identificativi univoci raccolti</string>
+    <string name="pref_exposure_collected_rpis_summary"><xliff:g example="63">%1$d</xliff:g> identificativi univoci nell\'ultima ora</string>
+    <string name="pref_exposure_advertising_id_title">Identificativi univoci attualmente trasmessi</string>
+    <string name="pref_exposure_app_last_report_title">Ultimo resoconto (<xliff:g example="3 hours ago">%1$s</xliff:g>)</string>
+    <string name="pref_exposure_app_last_report_summary_diagnosis_keys">Processate <xliff:g example="121031">%1$d</xliff:g> chiavi di diagnosi.</string>
+    <string name="pref_exposure_app_last_report_summary_encounters_no">Nessun incontro esposto al virus riscontrato.</string>
+    <string name="pref_exposure_app_last_report_summary_encounters_prefix">Riscontrati <xliff:g example="3">%1$d</xliff:g> incontri esposti:</string>
```

Comment:
I think it's better to translate "exposure encounter" into "esposizione" (so that it would be "Nessuna esposizione al vius riscontrata").

Reply:
That's far better, you are right